README.md

```diff
@@ -1,2 +1,17 @@
 # Pupil Labs Website
 
+## Dependencies & Set Up
+
+**Install node.js and npm on windows**
+
+1. Go to https://nodejs.org/en/ and install node.js (npm included)
+2. After installation restart computer
+3. Test node.js and npm
+	- open command line tool
+	- type `node -v` (should print out a version number)
+	- type `npm -v` (should print out a version number)
+4. In command line tool go to ../pupil-labs-website repository
+	- type `npm install`
```

Comment:
Revise to:
Install packages locally with `npm install` Packages are defined in `package.json`